app/components/views/Offset/index.tsx

```diff
@@ -464,54 +464,50 @@ export const Offset = (props: Props) => {
             labelAlignment="start"
           />
 
-          <div className={styles.input}>
-            <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.beneficiary">BENEFICIARY</Trans>
-              </Text>
-            </label>
-            <input
-              value={beneficiary}
-              onChange={(e) => {
-                setBeneficiary(e.target.value);
-              }}
-              placeholder={t({
-                id: "offset.who_beneficiary",
-                message: "To whom will this retirement be credited?",
-              })}
-            />
-          </div>
-          <div className={styles.input}>
-            <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.beneficiary_address">
-                  BENEFICIARY ADDRESS (optional: defaults to connected address)
+          <div className={styles.beneficiary}>
+            <Text t="caption" color="lighter">
+              <Trans id="offset.retirement_message">
```

Comment:
Duplicate id
Indeed I open the preview and I can see you need to run `extract-strings` as well, and that this is ignoring the wrapped text and using the pre-defined value in the `.po` file

It's annoying that this problem only surfaces once you are in preview mode, but everything seems to work fine in local dev mode. Probably because the .PO file is missing an id so it uses the fallback?
I'm going to open an issue and see if biwano can find a way to smooth this out so we either:
A. catch these mistakes earlier  (warn on duplicates?)
B. don't introduce them at all (no ids?)

Reply:
Nice catch 👀
Yeah I didn't catch that in my local dev environment.

Let's document that somewhere as part of the dev workflow. I'll add a it to the PR template as a prompt.